src/BackgroundPublish.h

```diff
@@ -72,16 +77,83 @@ class BackgroundPublish {
      * @param[in] flags PublishFlags type for the request
      * @param[in] priority priority of message. Lowest is highest priority, zero indexed
      * @param[in] cb callback on publish success or failure
+     *
+     * @return TRUE if request accepted, FALSE if not
+     */
+    bool publish(const char* name,
+                 const char* data = nullptr,
```

Comment:
What is the use case for a publish without the `data` argument?

Reply:
Looks like it's for an event where the event name itself is the only information?

Reply:
yeah, I guess that is supported